ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/ccache/tasks/main.yml

```diff
@@ -9,17 +9,14 @@
   tags: ccache
 
 - name: Download ccache.tar.gz
-  get_url:
-    url: https://www.samba.org/ftp/ccache/ccache-3.4.2.tar.gz
-    dest: /tmp/ccache.tar.gz
-    mode: 0440
-    validate_certs: False
+  shell: cd /tmp && wget https://www.samba.org/ftp/ccache/ccache-3.4.2.tar.gz
```

Comment:
Can you change this to `wget -O /tmp/ccache-3.4.2.tar.gz https://...` instead of the `cd /tmp && ...`

Reply:
Or even wget -O /tmp/ccache.tar.gz to avoid the need for the version to be listed multiple times in the file